packages/numpy/meta.yaml

```diff
@@ -18,16 +18,26 @@ build:
   # It breaks our test because there is a .pyc in the directory.
   post: |
     rm -rf numpy/distutils/__pycache__
+  # NumPy 2.1 disabled visibility for symbols outside of extension modules
+  # by default, so this breaks SciPy tests from modules that use f2py to
+  # build because they rely on the visibility of symbols in NumPy. This flag
+  # is currently used as a stop-gap measure. For more information, please see
+  # 1. https://github.com/numpy/numpy/pull/26286, and
+  # 2. https://github.com/numpy/numpy/pull/26103.
   cflags: |
+    -DNPY_API_SYMBOL_ATTRIBUTE=__attribute__((visibility("default")))
```

Comment:
Oh, you need to set this flag on the scipy side, not on the numpy side. Sorry, I wasn't clear.

Reply:
Thanks! For now, I'll downgrade to NumPy 2.0.2 and remove this completely, as you instructed in https://github.com/numpy/numpy/issues/27865#issuecomment-2503268541.